tests/test-stake-pool-sdk.ts

```diff
@@ -178,6 +180,81 @@ describe("test basic functionalities", () => {
       );
     });
 
+    it("it calcSolDeposit() and calcSolDepositsInverse() works correctly", async () => {
+      const socean = new Socean("testnet");
+      const stakePool = await socean.getStakePoolAccount();
+
+      const depositAmountSol = Math.random() / 4;
```

Comment:
nit: the `/ 4` in the `it calcSolDeposit() matches actual droplets received` test was because the previous test step stakes at most 0.25 SOL into the pool on testnet. You can remove this since we're just checking the results of the calculation here